README.md

```diff
@@ -44,6 +48,11 @@ If you are using Maven without the BOM, add this to your dependencies:
   <artifactId>google-cloud-storage</artifactId>
   <version>2.36.0</version>
 </dependency>
+<dependency>
```

Comment:
I think we want to change this to importing the `google-cloud-storage-bom` rather than using storage-control directly.

This will make it easier for customers when the library moves from alpha to beta to ga as they won't have to update their storage-control dependency, only `google-cloud-storage-bom`

Reply:
Actually, we probably want to do the same thing for our samples modules that aren't using `libraries-bom`

Reply:
this is for the sample that tells people how to use the library without the bom though?

see above:  `If you are using Maven without the BOM, add this to your dependencies:`

Reply:
I think that statement is meant to mean `libraries-bom`, we might need to disambiguate.

Reply:
I am going to release this as is but we can update it for the next release.